source/educators/how-tos/course_development/exercise_tools/setup_lti_store.rst

```diff
@@ -0,0 +1,163 @@
+.. _Set up a Reusable LTI Store:
+
+
+Set up a Reusable LTI Store
+###########################
+
+.. tags:: educator, how-to, lti
+
+The Reusable LTI Store centralizes LTI configuration in Django Admin.  
+Course authors then reference a single reusable configuration in Studio (via the LTI Consumer component).  
+A single configuration can be referenced multiple times, eliminating repeated copy/paste of tool credentials.
+
+
+Overview
+********
+
+The Reusable LTI Store allows operators and course authors to simplify setup by:
+
+* Defining LTI configurations once in Django Admin.
+* Allowing course authors to reference those configurations in Studio.
+* Reducing errors and duplication of tool credentials.
+
+This feature supports LTI 1.3 integrations and is particularly useful for institutions running multiple courses with the same tool.
+
+
+
+Before you start
+****************
+
+To use the Reusable LTI Store, ensure the following:
+
+* You have an operational Open edX deployment.
+* You have staff access to Django Admin (model-level access to ``lti_store``).
+* You have access to an LTI 1.3 tool with:
+
+  - Client ID
+  - Deployment ID
+  - OIDC/Authorization URL
+  - Launch URL
+  - JWKS (key set) URL or a tool public key
+
+* For Tutor-based deployments: You can install Tutor plugins.
+* For non-Tutor deployments: You can install and configure Django apps.
+
+
+
+Install the LTI Store Tutor Plugin
+**********************************
+
+1. Install the plugin from the `LTI Store Repository`_ and follow the plugin’s README to enable it for your environment.
+2. Rebuild and restart your platform. Apply the plugin’s documented Tutor config, images rebuild, and services restart steps.
+
+.. note::
+   For non-Tutor installations, follow `Non-Tutor Install Instructions`_. Most community operators use Tutor, but some large operators do not.
+
+
+
+Grant staff access for LTI Store administration
+***********************************************
+
+To create and manage reusable configurations:
+
+* Ensure the user has ``is_staff`` enabled.
+* Assign permissions for the LTI Store models only (no superuser needed).
+* This limits access to just the LTI Store configuration screens.
+
+
+Create a reusable LTI configuration
+***********************************
+
+.. note::
+   The following steps correspond to an **LTI 1.3 configuration**.
+
+1. Open Django Admin.
```

Comment:
```suggestion
#. Open Django Admin.
```

Use `#` signs instead of direct numbers, which will make it much easier to add/remove steps